### ISSUE TITLE
CMS, components, design system, sitewide  508-defect-2 [AXE-CORE, SEMANTIC MARKUP]: Landmarks MUST have a unique role or role/label/title (i.e. accessible name) combination

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8250]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**This issue is in both the Public Websites CMS templates and the VA design system components, so both teams would need to address.**

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

> **Please note:** This particular ticket is an issue with the CMS templates, and also exists throughout our React apps. The main navigation, side navigation, and breadcrumbs need distinct `aria-labels` to distinguish them for assistive technology. Ideally, this would also be fixed in our React apps.
> * main nav: aria-label="main"
> * side nav: aria-label="secondary"
> * breadcrumbs: aria-label="breadcrumb"

## Description

The main navigation and secondary navigation **must** have unique role/label/title to distinguish for assistive tech. This is a sitewide issue previously documented in the following issues, to be remediated by VSP:
* [#4215](https://github.com/department-of-veterans-affairs/va.gov-team/issues/4215) — fixed

* [#6368](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6368) — still open, 11/04/2020

* [#7723](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7723) — still open, 11/04/2020

## Point of Contact

**VFS Point of Contact:** Jennifer

**Comment by MarciMcGuireGCIO** [comment]
Moving this to a sitewide epic so that remainder of the original epic (https://github.com/department-of-veterans-affairs/va.gov-team/issues/8247) can be closed.

**Comment by MarciMcGuireGCIO** [comment]
Changing from Public Websites to Decision Tools team, Find a VA Form Product

**Comment by MarciMcGuireGCIO** [comment]
Related thread on Slack:  https://dsva.slack.com/archives/CUB5X5MGF/p1609794779352100

**Comment by zacharymorel** [comment]
I think this falls back on the PW team on the basis we don't upkeep the Site wide navigations menus.

**Comment by MarciMcGuireGCIO** [comment]
@brianalloyd @swirtSJW - could I also trouble you to review this 508 issue and let me know if it would be handled by PW or CMS?  Thx!

**Comment by MarciMcGuireGCIO** [comment]
Removing Decision Tools / Find a VA Form tag off this since the descriptions states it is to be remediated by VSP.
